middleware/auth.js

```diff
@@ -0,0 +1,29 @@
+const protectedRoutes = ['/account/index', '/account/list']
+
+const isAuthorised = (req, res, next) => {
+    console.log('isAuth', req.originalUrl)
+
+    if (!req.session.authorised && protectedRoutes.includes(req.originalUrl)) {
+        console.log('not authorised')
+        res.render('users/login', {
```

Comment:
Probably want to redirect in these middlewares - otherwise you could end up on something like `/dashboard` but it would show the login page